nrf_modem/doc/dectphy.rst

```diff
@@ -24,35 +24,34 @@ Regulations
 
 DECT NR+ operates on free but regulated radio channels. The availability of the channels and the exact regulation to use them varies in different countries.
 
-.. important::
-   It is the developer's responsibility to operate the devices according to the local regulation, in all situations.
+.. note::
+   It is your responsibility to operate the devices according to the local regulation, in all situations.
 
-.. warning::
+.. important::
    The European Telecommunications Standards Institute (ETSI) has published European Normative document `ETSI EN 301 406-2`_, which restricts operation on specific absolute channel frequency numbers within the available range.
-   If you plan to operate a device using the DECT NR+ PHY modem firmware in territories where ETSI EN 301 406-2 is applicable, you **must** refer to table 4.3.2.3-2 to see which channel frequency numbers you are allowed to use.
+   If you plan to operate a device using the DECT NR+ PHY modem firmware in territories where ETSI EN 301 406-2 is applicable, you must refer to table 4.3.2.3-2 to see which channel frequency numbers you are allowed to use.
 
-.. warning::
-	The Federal Communication Commission (FCC) rules and regulations for radio frequency devices are located in `47 CFR Part 15`_.
-	If you plan to operate a device using the DECT NR+ PHY modem firmware in territories where 47 CFR Part 15 is applicable, you **must** refer to `47 CFR Part 15.323`_ to see which channel frequency numbers you are allowed to use.
+   The Federal Communication Commission (FCC) rules and regulations for radio frequency devices are located in `47 CFR Part 15`_.
+   If you plan to operate a device using the DECT NR+ PHY modem firmware in territories where 47 CFR Part 15 is applicable, you must refer to `47 CFR Part 15.323`_ to see which channel frequency numbers you are allowed to use.
 
 Protocol stack overview
 ***********************
 
-The physical layer is only one part of the DECT-2020 New Radio protocol stack (DECT NR+).
+The physical layer is only one part of the DECT-2020 new radio protocol stack (DECT NR+).
 
 The whole DECT NR+ protocol stack consist of four layers:
 
-* PHY or physical layer
-* MAC or medium access layer, including co-existing and channel access operation
-* DLC or data link control, including routing, segmentation and re-assembly of messages
-* CVG or convergence layer, including message flow control, multiplexing and transmission service level control
+* Physical layer (PHY)
+* Medium Access Control (MAC) layer, including co-existing and channel access operation
+* Data link control (DLC), including routing, segmentation and re-assembly of messages
+* Convergence layer (CVG), including message flow control, multiplexing and transmission service level control
```

Comment:
There are new terms used in the documentation. These could be added to the glossary page.

Reply:
ok, will add